File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/CantanteFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.Cantante;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class CantanteFactoryTest {

    @Test
    void createCantanteTest() {
        Cantante Cantante = CantanteFactory.create();
        // Cantante no nulo, para ver si se han asignado correctamente los datos
        assertNotNull(Cantante);
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/CantanteFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionBandaMusica.models.Musico

internal class CantanteFactoryTest {
    @Test
    fun createCantanteTest() {
        val Cantante: Musico.Cantante = CantanteFactory.create()
        // Cantante no nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(Cantante)
    }
}